src/zcl/definition/dataType.ts

```diff
@@ -32,6 +32,7 @@ enum DataType {
     int32 = 43,
     enum8 = 48,
     enum16 = 49,
+    beUint24 = 50,
```

Comment:
Could you add this to https://github.com/Koenkk/zigbee-herdsman/blob/master/src/zcl/definition/buffaloZclDataType.ts instead (since this is not an official Zigbee type)

Reply:
The changes have been made and can be reviewed now. Thank you for bringing this to my attention